### ISSUE TITLE
Generated output, possible error in code

### ISSUE THREAD

**Issue by vkeswani** [fabrahman/ReBART#3]
Hi authors

1. Could you provide a requirements.txt?
2. There might be an error in encoder_decoder.py in the inputs/outputs (lines 43-70). My guess is the text is being encoded twice.
3. Could you provide an example of how the predictions from generate.py look?

Thanking you in anticipation, really appreciate your work!

**Comment by fabrahman** [comment]
Hi @vkeswani 
1. Here is the [requirements](https://github.com/fabrahman/ReBART/blob/main/requirements.txt) file.
2. Can you please copy/paste the full error here so that we can reproduce? Also what's your transformer version? you should use 3.0.0

**Comment by vkeswani** [comment]
Hi @fabrahman 

Thanks a lot for your response. It turned out to be a transformers version issue. The code is working with 3.0.0. 
I performed experiments with a couple of datasets and was able to get results closer to those in the paper. 

In evaluation.py (line 112), I guess 'id_[2:-1]' would be replaced by 'id_'. 
I was able to run all the steps smoothly with the suggested versions in the requirements file.

Thanks a lot for your help!

Regards
Vishal
